.travis.yml

```diff
@@ -17,11 +23,13 @@ env:
 virtualenv:
   system_site_packages: true
 
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+
 install:
   - git clone https://github.com/OCA/reporting-engine.git ${HOME}/reporting-engine -b ${VERSION}
```

Comment:
Is this repos already cloned by maintainer-quality-tools since it's declared in  https://github.com/OCA/account-financial-reporting/blob/8.0/oca_dependencies.txt?